hashintel/hash
# Move workspace homepage to `/`

## 🌟 What is the purpose of this PR?




This PR moves the workspace homepage to the `/` route.

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203363157432081/1203491213980896/f) _(internal)_

## 🔍 What does this change?




See commits.

## 📜 Does this require a change to the docs?



Not that I'm aware of.

## ⚠️ Known issues



## 🐾 Next steps



- [Move page route from `/<accountId>/<pageEntityUuid>` to `/@shortname/<pageEntityUuid>`](https://app.asana.com/0/1203363157432081/1203491213980897/f)

## 🛡 What tests cover this?



Manual tests

## ❓ How to test this?



1. Run the HASH App locally and log in
2. Observe the homepage is located at `/`

## 📹 Demo
